### PR TITLE
ENTERPRISE-344 : Claims Status Endpoint General Documentation Updates

### PR DESCRIPTION
**Pull request by wootencl**
Completion of [ENTERPRISE-344](https://pokitdok.atlassian.net/browse/ENTERPRISE-344) ticket which entailed the updating of the claims status API endpoint general documentation.

Fixing a misspelling in the authorization endpoint example response json.

### CODE REVIEW

File: source/includes/_claimstatus.md [hunk 1]
```diff
@@ -1043,55 +1037,144 @@ submitting a claim to check its status.
...
-| patient.birth_date         | The patient’s birth date as specified on their policy.                                                                               |
-| patient.id                 | The patient’s member identifier.                                                                                                     |
-| patient.first_name         | The patient’s first name as specified on their policy.                                                                               |
-| patient.last_name          | The patient’s last name as specified on their policy.                                                                                |
-| provider.first_name        | The provider’s first name when the provider is an individual.                                                                        |
-| provider.last_name         | The provider’s last name when the provider is an individual.                                                                         |
-| provider.npi               | The NPI for the provider.                                                                                                            |
```
Comment (Dr-Mantis-Toboggan) [inline source/includes/_claimstatus.md]: Need to specify that it must be the billing provider's NPI. "The NPI for the billing provider"

File: source/includes/_claimstatus.md [hunk 2]
```diff
@@ -1043,55 +1037,144 @@ submitting a claim to check its status.
...
+| provider.middle_name                     | The provider’s middle name when the provider is an individual.                                                                                                                   |
+| provider.last_name                       | The provider’s last name when the provider is an individual.                                                                                                                     |
+| provider.suffix                          | The provider’s suffix when the provider is an individual.                                                                                                                        |
+| provider.tax_id                          | The provider’s tax identifier.                                                                                                                                                   |
+| provider.npi                             | The NPI for the provider.                                                                                                                                                        |
```
Comment (Dr-Mantis-Toboggan) [inline source/includes/_claimstatus.md]: Need to specify that its the NPI of the billing provider. "The NPI for the billing provider"

File: source/includes/_claimstatus.md [hunk 3]
```diff
@@ -1043,55 +1037,144 @@ submitting a claim to check its status.
...
 | patient.claims.adjudication_finalized_date            | The date adjudication was finalized for the claim.                                                                                                                                                                                                                                                                                                 |
 | patient.claims.applied_to_deductible                  | Boolean that indicates whether or not claim charges are applied to the deductible.                                                                                                                                                                                                                                                                 |
+| patient.claims.patient_control_number                 | The control number associated with a patient.                                                                                                                                                                                                                                                                                                      |
```
Comment (Dr-Mantis-Toboggan) [inline source/includes/_claimstatus.md]: The control number associated with the a patient. This number is assigned by the payer.

File: source/includes/_claimstatus.md [hunk 4]
```diff
@@ -1043,55 +1037,144 @@ submitting a claim to check its status.
...
+| provider.organization_name               | The provider’s name when the provider is an organization. first_name and last_name should be omitted when sending organization_name.                                             |
+| service_date                             | The date services were performed or started for the claim service period.                                                                                                        |
+| service_end_date                         | Optional: The date services ended for the claim service period.                                                                                                                  |
+| subscriber                               | Optional: The subscriber associated with the claim. Specify when the patient is not the subscriber. Uses the member [object](#claims_status_member_object).                      |
+| tracking_id                              | Optional: The payer's claim tracking id. Specify a tracking id to refine the search criteria for a specific claim.                                                               |
```
Comment (bstmartin) [inline source/includes/_claimstatus.md]: Suggested wording for tracking_id in request parameters - "Optional: The payer's claim tracking id. Specify this value to refine the search criteria for a claim.  If a payer claim control number was returned on the claim submission, this number should be used in the tracking id field for the greatest chance of returning valid claim status information."

**Pull request merged**